journald/journald.hpp

```diff
@@ -170,6 +187,10 @@ struct Flags : public virtual flags::FlagsBase
   Option<std::string> logrotate_filename;
   std::string logrotate_path;
   Option<std::string> user;
+
+  // Only used if the `destination_type` has `fluentbit`.
```

Comment:
I would not use backticks for the value (fluentbit) here, instead single or doublequotes.